.github/workflows/publish.yml

```
@@ -0,0 +1,172 @@
+    name: Pre-CI (Extract Commit Message)
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    outputs:
+      commit-message: ${{ steps.get_commit_message.outputs.commit-message }}
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - id: get_commit_message
+        run: |
```

actionlint (1.7.8)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)